newIDE/app/src/MainFrame/index.js

```diff
@@ -1440,6 +1475,7 @@ class MainFrame extends React.Component<Props, State> {
               }
               onRenameExternalEvents={this.renameExternalEvents}
               onSaveProject={this.save}
+              onSaveAsProject={this.saveAs}
```

Comment:
You're passing a prop to ProjectManager, but this prop is not expected by ProjectManager.
You can remove it, it's useless unless you add a "save as" button in the project manager - which is useless because it's only for the web-app :)

Reply:
The "prop" you have defined is only here to tell to the Flow static type checker what are the props that must be passed when you write `<MainFrame ...>`. i.e: in LocalApp.js and BrowserApp.js